src/classes/metrics.py

```diff
@@ -64,8 +64,8 @@
         # Get the distro name and version (if any)
         linux_distro = "-".join(platform.linux_distribution())
 
-except Exception as Ex:
-    log.error("Error determining OS version in metrics.py")
+except Exception:
+    log.debug("Error determining OS version in metrics.py")
```

Comment:
I have no idea why this names its file. It shouldn't. Like I said, that info is automatically included.
```suggestion
    log.debug("Error determining OS version", exc_info=1)
```